ultra/README.md

```diff
@@ -1,89 +1,40 @@
 # ULTRA
 
-Unprotected Left Turn using Reinforcement-learning Agents
----
+## Unprotected Left Turn using Reinforcement-learning Agents
+
 Ultra provides a gym-based environment using SMARTS for tackling intersection navigation and more specifically unprotected left turn.
 
 Here is the summary of key features:
- - creating customized scenarios with different levels of difficulty
- - defining/analyzing traffic designs including low-mid-high densities
- - defining/analyzing social-vehicle behaviors
- - configurable train/test parameters
- - train/test custom RL algorithms against benchmark results
-
+ - Creating customized scenarios with different levels of difficulty.
+ - Defining and analyzing traffic designs including low-mid-high densities.
+ - Defining and analyzing social-vehicle behaviors.
+ - Configurable train and test parameters.
+ - Train and test custom RL algorithms against benchmark results.
```

Comment:
Trying to rewrite for features rather than actions
```suggestion
 - Customizable scenarios with different levels of difficulty.
 - Analysis tools to evaluate traffic designs including low-mid-high densities.
 - Tools to support analyzing social-vehicle behaviors.
 - Configurable train and test parameters.
 - Benchmark results to train and test custom RL algorithms against.
```

Reply:
Oh perfect, thanks!